src/patterns/problem-with-the-service-pages/index.md.njk

```diff
@@ -9,6 +9,8 @@ layout: layout-pane.njk
 status: Experimental
 statusMessage: This pattern is currently experimental because <a class="govuk-link" href="#research-on-this-pattern">more research</a> is needed to validate it.
 ---
+This guidance is for government teams building online services. [To find information and services for the public, go to GOV.UK.](https://www.gov.uk/)
+
 {% from "_example.njk" import example %}
 
 This is a page that tells someone there is something wrong with the service. They are also known as 500 and internal server error pages.
```

Comment:
Tried to simplify this sentence, especially amongst the other 2 as Eoin's flagged, by focussing on the outcome of this pattern (tell users) instead of making another reference to the page or pattern.

```suggestion
Tell the user when there is something wrong with the service. These are also known as 500 and internal server error pages.
```